pelican-gfm/gfmSetup.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python3 -B
+import sys
+import os
+import subprocess
+import shutil
+import tarfile
+import tempfile
+import Settings
+
+# Eventually, equivalents for
+# other operating systems / package
+# managers could be written for this
+
+
+def dpkg_installed(package):
+    """ Uses Dpkg to determine whether or not a package is installed
+    requires: <package name>"""
+    t1 = subprocess.Popen(["dpkg", "-l"], stdout=subprocess.PIPE)
+    t2 = subprocess.Popen([
+                            "grep",
+                            "-q",
+                            package
+                          ], stdout=subprocess.PIPE, stdin=t1.stdout)
```

Comment:
So... you're running `grep` to check for a string in a bunch of lines *from `python`*? :)

Reply:
yeah, that does seem silly. thanks for pointing that out.